CHANGELOG.md

```diff
@@ -2,6 +2,14 @@
 
 All notable changes to the "azure-cosmosdb" extension will be documented in this file.
 
+## 0.13.0 - 2020-05-28
+### Added
+- PostgreSQL support
+  - Basic CRUD support added to PostgreSQL databases and integrated with the exisitng extension; updates to Tables is not yet supported
```

Comment:
What about this: "Create and view PostgreSQL servers, databases, and functions"

I'm not a huge fan of using "CRUD". I don't know if this is just me, but I didn't know what it meant for a long time 😂 And we probably don't need to list *all* of the possible actions you can take on a resource

Reply:
Whenever I hear CRUD, I think 💩